packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -41,12 +41,18 @@ export const SimpleDocument: FC<Props> = ({
   let html,
     passage: QueryResultPassage | null = null;
   if (document) {
-    // JSON files will default to displaying the specified body field, `text` field, or passage highlighting field,
+    // JSON and CSV files will default to displaying the specified body field, `text` field, or passage highlighting field.
     // Otherwise an error is shown
     const isJsonType = isJsonFile(document);
+    const isCsvType = isCsvFile(document);
```

Comment:
To make this a bit cleaner, just put both into a single var:

```suggestion
    const isCsvOrJson = isCsvFile(document) || isJsonFile(document);
```